src/eduid_userdb/support/db.py

```diff
@@ -75,27 +69,19 @@ def get_authn_info(self, user_id):
             doc = self.model(doc)
         return doc
 
-
-class SupportVerificationsDB(BaseDB):
-
-    model = models.UserVerifications
-
-    def __init__(self, db_uri):
-        db_name = 'eduid_dashboard'
-        collection = 'verifications'
-        super(SupportVerificationsDB, self).__init__(db_uri, db_name, collection)
-
-    def get_verifications(self, user_id):
+    def get_credential_info(self, credential_id):
         """
-        :param user_id: User objects user_id property
-        :type user_id: ObjectId | str | unicode
-        :return: A list of dict
-        :rtype: list
+        :param credential_id: Credential id
+        :type credential_id: ObjectId | str | unicode
+        :return:  A document dict
+        :rtype: dict
         """
-        if not isinstance(user_id, ObjectId):
-            user_id = ObjectId(user_id)
-        docs = self._get_documents_by_attr('user_oid', user_id, raise_on_missing=False)
-        return [self.model(doc) for doc in docs]
+        if not isinstance(credential_id, ObjectId):
+            user_id = ObjectId(credential_id)
```

Comment:
this should be credential_id = ObjectId(credential_id), right?

Reply:
Right!